packages/core/src/storybook/components/related-components/descriptions/dialog-description/dialog-description.tsx

```diff
@@ -43,7 +43,7 @@ export const DialogDescription = () => {
     <RelatedComponent
       component={component}
       title="Dialog"
-      href="/?path=/docs/components-dialog-dialog--docs"
+      href="/?path=/docs/popover-dialog--docs"
```

Comment:
Just an idea to explore some day
https://monday.monday.com/boards/3532714909/pulses/6426609360
addon-links should have something to handle it

Reply:
We also have the StorybookLink component, it uses Storybook's API, we can use it as well

Reply:
didn't we said we should deprecate it?

Reply:
and yep, it uses `import { hrefTo } from '@storybook/addon-links';`, I believe you've added it

Reply:
Yes
We talked about deprecating Link in favor of LinkComponent

Reply:
> and yep, it uses `import { hrefTo } from '@storybook/addon-links';`, I believe you've added it

Yes exactly https://github.com/mondaycom/monday-ui-react-core/blob/docs/fix-dialog-link-in-related-components/packages/storybook-blocks/src/components/storybook-link/storybook-link.tsx#L3

Reply:
Awesome, will add to the ticket this thread